azure-pipelines/pull-request-validation/pr-msal.yml

```diff
@@ -0,0 +1,74 @@
+# File: azure-pipelines\pull-request-validation\pr-msal.yml
+# Description: Assemble and run unit test
+# Variable: 'ENV_VSTS_MVN_ANDROIDADACCOUNTS_USERNAME' was defined in the Variables tab
+# Variable: 'mvnAccessToken' was defined in the Variables tab
+# https://docs.microsoft.com/en-us/azure/devops/organizations/accounts/use-personal-access-tokens-to-authenticate
+name: $(date:yyyyMMdd)$(rev:.r)
+
+variables:
+- name: BuildParameters.jdkVersion
+  value: 1.8
+- name: BuildParameters.jdkArchitecture
+  value: x64
+- name: BuildParameters.javaHomeSelection
+  value: JDKVersion
+- name: robolectricSdkVersion
+  value: 28
+
+trigger:
+  branches:
+    include:
+    - dev
+    - master
+  batch: True
+
+resources:
+  repositories:
+  - repository: common
+    type: github
+    name: AzureAD/microsoft-authentication-library-common-for-android
+    ref: dev
+    endpoint: ANDROID_GITHUB
+
+jobs:
+- job: Phase_1
+  displayName: Phase 1
+  cancelTimeoutInMinutes: 1
+  pool:
+    name: Hosted Windows 2019 with VS2019
+  steps:
+  - checkout: self
+    clean: true
+    submodules: recursive
+    persistCredentials: True
+  - template: azure-pipelines/templates/steps/credscan-policheck.yml@common
```

Comment:
this does use v2, right?

Reply:
yes it does
https://github.com/AzureAD/microsoft-authentication-library-common-for-android/blob/5fbf67bd65f7a122a098328438877ce64646ebf5/azure-pipelines/templates/steps/credscan-policheck.yml#L10